Scripts/Source/minai_Sex.psc

```diff
@@ -274,8 +274,32 @@ Function ActionResponse(actor akTarget, actor akSpeaker, string sayLine, actor[]
     StartSexOrSwitchTo(akSpeaker, akTarget, Player, bPlayerInScene, "blowjob")
   elseif stringutil.Find(sayLine, "-handjob-") != -1
     StartSexOrSwitchTo(akSpeaker, akTarget, Player, bPlayerInScene, "handjob")
-  elseIf stringutil.Find(sayLine, "-groupsex-") != -1 || stringUtil.Find(sayLine, "-orgy-") != -1 || stringUtil.Find(sayLine, "-threesome-") != -1 || stringUtil.Find(sayLine, "-fuck-") != -1
+  elseif stringutil.Find(sayLine, "-footjob-") != -1
+    StartSexOrSwitchTo(akSpeaker, akTarget, Player, bPlayerInScene, "footjob")
```

Comment:
All of these tags are the OStim specific tags. Let's use the sexlab tags in the code everywhere, and convert them to ostim tags in the tag converter function.

Reply:
Ok, made those changes.